ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -2355,6 +2355,42 @@ def test_paddle_tensor_numel(
     )
 
 
+# pow
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="pow",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Could you please change this to `get_dtypes("valid")`? The helper will sample the correct dtypes according to the `@with_supported_dtypes` which you have added to the function.

Other that that, looks good to me!

Reply:
Oh you will need to resolve the conflicts as well for me to be able to merge the PR. Thanks :)